tests/helper_util.py

```diff
@@ -53,6 +52,8 @@ def do_pickle(obj, func=(lambda x : x), copyable=True, reprable=True):
                         obj = eval(repr(obj)).  This is often a desirable property of objects,
                         but it may not always be possible (or efficient). [default: True]
     """
+    import coord
+
```

Comment:
I 100% agree this is safe for coord. I think I didn't put enough context in the PR description.

This change is so that the test suite interacts nicely with the jax-galsim test suite. The jax-galsim test suite, instead of vendoring the tests from coord and galsim, uses the upstream test suites. It replaces all imports of galsim and coord with imports of jax_galsim. This helper file gets reimported after the initial replacement and so breaks the replacement. The change here hides the import so things do not break.

Reply:
Can you put a comment to that effect in the code then to explain the non-standard import.  (Else I might break it for you sometime down the line, forgetting about this.)